package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@contractbook/puppeteer-pdf",
   "author": "Patryk Domałeczny <patryk.domaleczny@gmail.com>",
-  "version": "1.4.0",
+  "version": "1.6.0",
```

Comment:
We need to remember to push the new tag version and update that in FS images :D

Reply:
Yep, I'm in progress with this one :) Once this is in master, I'll update the tag.